LockboxXCUITests/LockBoxScreenGraph.swift

```diff
@@ -201,102 +213,57 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
+    func skipAutofillConfiguration() {
+        if #available(iOS 12.0, *) {
+            //If autofill is set this option will not appear
+            sleep(5)
+            if (app.buttons["setupAutofill.button"].exists) {
+                app.buttons["notNow.button"].tap()
             }
-            group.wait()
         }
     }
 
-    func completeVerification(uid: String, code: String, done: @escaping () -> ()) {
-        // POST to EndPoint api.accounts.firefox.com/v1/recovery_email/verify_code
-        let restUrl = URL(string: postEndPoint)
-        var request = URLRequest(url: restUrl!)
-        request.setValue("application/json; charset=UTF-8", forHTTPHeaderField: "Content-Type")
-
-        request.httpMethod = "POST"
-
-        let jsonObject: [String: Any] = ["uid": uid, "code":code]
-        let data = try! JSONSerialization.data(withJSONObject: jsonObject, options: JSONSerialization.WritingOptions.prettyPrinted)
-        let json = NSString(data: data, encoding: String.Encoding.utf8.rawValue)
-        if let json = json {
-            print("json \(json)")
-        }
-        let jsonData = json?.data(using: String.Encoding.utf8.rawValue)
-
-        request.httpBody = jsonData
-        print("json \(jsonData!)")
-        let task = URLSession.shared.dataTask(with: request) { data, response, error in
-            if let error = error {
-                print("error:", error)
-                return
-            }
-            done()
-        }
-        task.resume()
+    func tapOnFinishButton() {
+        waitforExistence(app.buttons["finish.button"])
+        app.buttons["finish.button"].tap()
     }
 
-    func verifyAccount(done: @escaping () -> ()) {
-        // GET to EndPoint/mail/test-9876@restmail.net
-        let restUrl = URL(string: getEndPoint)
-        var request = URLRequest(url: restUrl!)
-        request.httpMethod = "GET"
-
-        let task = URLSession.shared.dataTask(with: request) { data, response, error in
-            if(error != nil) {
-                print("Error: \(error ?? "Get Error" as! Error)")
-            }
-            let responseString = String(data: data!, encoding: .utf8)
-            print("responseString = \(String(describing: responseString))")
-
-            let regexpUid = "(uid=[a-z0-9]{0,32}$?)"
-            let regexCode = "(code=[a-z0-9]{0,32}$?)"
-            if let rangeUid = responseString?.range(of:regexpUid, options: .regularExpression) {
-                uid = String(responseString![rangeUid])
-            }
-
-            if let rangeCode = responseString?.range(of:regexCode, options: .regularExpression) {
-                code = String(responseString![rangeCode])
-            }
-
-            if (code != nil && uid != nil) {
-                let codeNumber = self.getPostValues(value: code)
-                let uidNumber = self.getPostValues(value: uid)
+    func waitForLockboxEntriesListView() {
+        waitforExistence(app.navigationBars["firefoxLockbox.navigationBar"])
```

Comment:
nice, I really like this function!! I think sync is getting kicked off slowly with some of the rust changes and this is a good workaround.